web/styles/app_variables.css

```diff
@@ -570,7 +570,8 @@
     /* .display-type and .list-toggler-container
        in subscriptions and user group settings overlays */
     --settings-overlay-subheader-height: 3.1428em; /* 44px at 14px/em */
-    --settings-overlay-header-button-height: 2em;
+    --settings-overlay-header-left-button-height: 2em;
+    --settings-overlay-header-right-button-height: 2.6429em; /* 37px at 14px/em */
```

Comment:
It might be useful to note why we have these two different button heights, and why their values differ.

Reply:
Turns out I'll just make them the same